zebedee-cms/src/main/java/com/github/onsdigital/zebedee/session/store/SessionsThreadLocal.java

```diff
@@ -0,0 +1,10 @@
+package com.github.onsdigital.zebedee.session.store;
+
+import com.github.onsdigital.impl.UserDataPayload;
+
+import javax.servlet.http.HttpServletRequest;
+
+public interface SessionsThreadLocal {
```

Comment:
Add javadoc to give a brief explanation of the purpose of this interface.

Reply:
added javadoc